README.md

```diff
@@ -27,14 +28,76 @@ cat > .env << __EOF__
 POSTGRES_DB=brigadier
 POSTGRES_USER=brigadier
 POSTGRES_PASSWORD=secret
-POSTGRES_HOST=localhost
+POSTGRES_HOST=0.0.0.0:8000
 POSTGRES_PORT=5433
-DJANGO_DEBUG=True
 __EOF__
 ```
-* Export env vars from env-file:
+* Create and run PostgreSQL docker container:
 ```shell
-export $(cat .env)
+docker run -d --name brigadier-postgres --hostname brigadier-postgres \
+-p 5433:5432 \
+--env-file .env \
+postgres:13.2-alpine
+```
+* Pull the uwsgi-docker image:
+```shell
+docker pull avo888/brigadier-uwsgi:latest
+```
+* Create .env-uwsgi file. Example:
+```shell
+cat > .env-uwsgi << __EOF__
+POSTGRES_HOST=brigadier-postgres
+POSTGRES_DB=brigadier
+POSTGRES_PASSWORD=secret
+POSTGRES_PORT=5432
+POSTGRES_USER=brigadier
+PS1=%n@%m %~ %%
+__EOF__
+```
+* Run the uwsgi-docker image:
+```shell
+docker run -d --name brigadier-uwsgi --hostname brigadier-uwsgi \
+-p 8000:8000 \
+--env-file brigadier/.env-uwsgi \
+--link brigadier-postgres \
+avo888/brigadier-uwsgi:latest
+```
+* Create objects from admin console.
+    1. Go to the browser and type '127.0.0.1:8000/admin'
+    2. Login
+    3. Create objects
+* Profit
+
+## Develop
+* Clone the rep:
+```shell
+git clone git@github.com:avoevodin/brigadier.git
+```
+* Create the venv:
+```shell
+python3 -m venv venv
+```
+* Activate the venv:
+```shell
+source venv/bin/activate
+```
+* Install requirements:
+```shell
+pip install -r requirements.txt
+```
+* Open brigadier-project directory
+```shell
+cd brigadier
+```
+* Create the .env file. Example:
+```shell
+cat > .env << __EOF__
+POSTGRES_DB=brigadier
+POSTGRES_USER=brigadier
+POSTGRES_PASSWORD=secret
+POSTGRES_HOST=0.0.0.0:8000
+POSTGRES_PORT=5433
+__EOF__
```

Comment:
* it's development section, should use DJANGO_DEBUG=True
* forget about django-debug-toolbar

Reply:
tutorial for django-debug-toolbar is in another file.